expression/chunk_executor.go

```diff
@@ -64,6 +64,100 @@ func VectorizedExecute(ctx sessionctx.Context, exprs []Expression, iterator *chu
 	return nil
 }
 
+func evalOneVec(ctx sessionctx.Context, expr Expression, input *chunk.Chunk, output *chunk.Chunk, colIdx int) error {
+	ft := expr.GetType()
+	result := output.Column(colIdx)
+	switch ft.EvalType() {
+	case types.ETInt:
+		if err := expr.VecEvalInt(ctx, input, result); err != nil {
+			return err
+		}
+		if ft.Tp == mysql.TypeBit {
+			i64s := result.Int64s()
+			buf := chunk.NewColumn(ft, input.NumRows())
+			buf.ReserveBytes(input.NumRows())
+			uintBuf := make([]byte, 8)
+			for i := range i64s {
+				if result.IsNull(i) {
+					buf.AppendNull()
+				} else {
+					buf.AppendBytes(strconv.AppendUint(uintBuf[:0], uint64(i64s[i]), 10))
+				}
+			}
+			// TODO: recycle all old Columns returned here.
+			output.SetCol(colIdx, buf)
+		} else if mysql.HasUnsignedFlag(ft.Flag) {
+			i64s := result.Int64s()
```

Comment:
We can do some tricks to avoiding copy data here since the underlying bytes of int64/uint64 are the same. (Please make sure it has test cases covered.)

Reply:
updated, PTAL

Reply:
See https://play.golang.org/p/ZVT9L_X05u8, so we can do a no-op here and just leave a comment.

Reply:
So magic... It seems you are right.
Now it is updated to a no-op.